src/main/java/net/datafaker/sequence/FakeStream.java

```diff
@@ -25,6 +26,11 @@ public boolean isInfinite() {
         return maxLength < 0;
     }
 
+    @Override
+    public Iterator<T> iterator() {
+        return get().iterator();
+    }
+
```

Comment:
Probably it's better to make such change on FakeSequence level like
```java
    public abstract <Sequence extends Iterable<T>> Sequence get();

    public final Iterator<T> iterator() {
        return get().iterator();
    }
```
and `FakeStream`, `FakeCollection` keep untouched

Reply:
@snuyanzin we can do that, but in that case we have to use Reflection API, as Stream doesn't extend Iterable interface. (It has it's own method with the same name...)
Something like this:
```java
    @Override
    public Iterator<T> iterator() {
        try {
            Object iterable = get();
            Method iterator = iterable.getClass().getMethod("iterator");
            iterator.setAccessible(true);
            //noinspection unchecked
            return (Iterator<T>) iterator.invoke(iterable);
        } catch (Exception e) {
            throw new RuntimeException(e);
        }
    }
```

Is this fine?

Reply:
true, ok then let's go with the initial version

Reply:
Ok, I'm leaving this unchanged